### DISCUSSION TITLE
How do I integrate this library into vue virtual scroll, I try many ways, but I doesn't work

### DISCUSSION THREAD

**Discussion by toandji** [idiotWu/smooth-scrollbar#547] [Q&A]
Here is my code : 
 <DynamicScroller
            :items="filteredTemplates"
            class=" w-full h-full bg-[#fbfafb] snap-y overflow-hidden overflow-y-auto"
            :min-item-size="375" 
            :itemSecondarySize="400"
            listClass="template-list"
            :updateInterval="0" 
            :typeField="'templateCode'"
            page-mode
            id="homepageRef"
          >
            <template v-slot="{ item, index, active }">
              <DynamicScrollerItem
                :item="item"
                :active="active"
                :size-dependencies="[
                  item.groups,
                ]"
                :data-index="index"
              >
                  <div :class="[(index == 0) ? 'bg-white rounded-b-xl' : 'pt-3', 'w-full template-item-wraper']">
                    <component :data="item" :class="`${item.id}-${item.templateCode} ${item.playCateName}`"
                      :playCateName="activeInfo" :is="item.templateCode">
                    </component>
                  </div>
              </DynamicScrollerItem>
            </template>
          </DynamicScroller>
 Then I wrapped this component like this : 
 const homepageRef = document.querySelector('#homepageRef');
    console.log("homepageRef", homepageRef)
    if (this.filteredTemplates && homepageRef) {
      Scrollbar.init(homepageRef, {damping: 0.5, renderByPixels: true, wheelEventTarget: (e) => console.log("e", e)});
    }

**Comment by sadeghbarati** [comment]
Check this code 👇  to understand how to implement your `smooth-scrollbar`

https://www.npmjs.com/package/@smooth-scrollbar-contrib/vue-test
